administrator/language/en-GB/plg_content_responsiveimages.sys.ini

```diff
@@ -0,0 +1,7 @@
+; Joomla! Project
+; (C) 2010 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_CONTENT_RESPONSIVEIMAGES="Content - Responsive Images"
+PLG_CONTENT_RESPONSIVEIMAGES_XML_DESCRIPTION="This plugin implements the responsive images with art direction functionality."
```

Comment:
Note that it is a good style to always put the newline as a last character if it is allowed by the file format.